lint

```diff
@@ -18,18 +18,13 @@
 set -e
 
 IGNORE_LINT_COMMENT=
-IGNORE_TEST_PACKAGES=
 IGNORE_SPELLINGS=
 while true; do
     case "$1" in
         -nocomment)
             IGNORE_LINT_COMMENT=1
             shift 1
             ;;
-        -notestpackage)
```

Comment:
Well, there are too many repos to update manually (see https://github.com/search?p=1&q=org%3Aweaveworks+-notestpackage&type=Code ) so I will make it a noop for now